server/remote_cache/hit_tracker/hit_tracker.go

```diff
@@ -75,7 +75,7 @@ const (
 	// New counter types go here!
 )
 
-func DetailedStatsEnabled() bool {
+var DetailedStatsEnabled = func() bool {
 	return *detailedStatsEnabled
 }
```

Comment:
nit: if you can use flags.Set here then we can make these functions again. They are probably functions to prevent the exact tampering that this CL is doing :P

Reply:
Reverted these.